.github/workflows/build-and-test.yaml

```
@@ -33,3 +33,12 @@ jobs:
 
       - name: Run cargo test
         run: cargo test
+
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install cargo-hack
+        uses: taiki-e/install-action@cargo-hack
+      - run: cargo hack build --all --rust-version
+      - run: cargo hack build --all --no-default-features --rust-version
```

Let's give the actual commands a different name.

```suggestion
      - name: Check MSRV compatibility
        run: |
           cargo hack build --all --rust-version
           cargo hack build --all --no-default-features --rust-version
```